packages/ember-htmlbars/tests/attr_nodes/class_test.js

```diff
@@ -35,112 +35,3 @@ QUnit.test('class renders before didInsertElement', function() {
-  view = EmberView.create({
-    context: { classes: 'large blue' },
-    template: compile('<div class={{classes}}></div>')
-  });
-  appendView(view);
-
-  equalInnerHTML(view.element, '<div class="large blue"></div>',
-                 'attribute is output');
-  ok(view.$('.large')[0], 'first class found');
-  ok(view.$('.blue')[0], 'second class found');
-});
-
-QUnit.test('class property is removed when updated with a null value', function() {
-  view = EmberView.create({
-    context: { class: 'large' },
-    template: compile('<div class={{class}}></div>')
-  });
-  appendView(view);
-
-  equal(view.element.firstChild.className, 'large', 'attribute is output');
-
-  run(view, view.set, 'context.class', null);
-
-  equal(view.element.firstChild.className, '', 'attribute is removed');
-});
-
-QUnit.test('class attribute concats bound values', function() {
-  view = EmberView.create({
-    context: { size: 'large', color: 'blue' },
-    template: compile('<div class=\'{{size}} {{color}} round\'></div>')
-  });
-  appendView(view);
-
-  strictEqual(view.element.firstChild.className, 'large blue round', 'classes are set');
-});
-
-QUnit.test('class attribute accepts nested helpers, and updates', function() {
-  view = EmberView.create({
-    context: {
-      size: 'large',
-      hasColor: true,
-      hasShape: false,
-      shape: 'round'
-    },
-    template: compile('<div class=\'{{if true size}} {{if hasColor \'blue\'}} {{if hasShape shape \'no-shape\'}}\'></div>')
-  });
-  appendView(view);
-
-  strictEqual(view.element.firstChild.className, 'large blue no-shape', 'classes are set');
-
-  run(view, view.set, 'context.hasColor', false);
-  run(view, view.set, 'context.hasShape', true);
-
-  strictEqual(view.element.firstChild.className, 'large  round', 'classes are updated');
-});
-
-QUnit.test('class attribute can accept multiple classes from a single value, and update', function() {
-  view = EmberView.create({
-    context: {
-      size: 'large small'
-    },
-    template: compile('<div class=\'{{size}}\'></div>')
-  });
-  appendView(view);
-
-  strictEqual(view.element.firstChild.className, 'large small', 'classes are set');
-
-  run(view, view.set, 'context.size', 'medium');
-
-  strictEqual(view.element.firstChild.className, 'medium', 'classes are updated');
-});
-
-QUnit.test('class attribute can grok concatted classes, and update', function() {
-  view = EmberView.create({
-    context: {
-      size: 'large',
-      prefix: 'pre-pre pre',
-      postfix: 'post'
-    },
-    template: compile('<div class=\'btn-{{size}} {{prefix}}-{{postfix}}    whoop\'></div>')
-  });
-  appendView(view);
-
-  strictEqual(view.element.firstChild.className, 'btn-large pre-pre pre-post    whoop', 'classes are set');
-
-  run(view, view.set, 'context.prefix', '');
-
-  strictEqual(view.element.firstChild.className, 'btn-large -post    whoop', 'classes are updated');
-});
-
-QUnit.test('class attribute stays in order', function() {
```

Comment:
Hm, the original test explicitly claim to care about ordering – and the new test does not (via the `classes` helper which sorts them). Like I mentioned above, I don't think we have a reason to care, but if for some reason we have I think the test will still pass (you would use a string instead of `classes` which does an exact match). @rwjblue can we drop this?